static/app/views/organizationIntegrations/abstractIntegrationDetailedView.tsx

```diff
@@ -266,7 +272,7 @@ class AbstractIntegrationDetailedView<
                   title={t(
                     'You must be an organization owner, manager or admin to install this.'
                   )}
-                  disabled={hasAccess}
+                  disabled={hasAccess || !this.requiresAccess}
```

Comment:
Don't show the tooltip if the user has access or the integration doesn't require it